File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/JAVA/OddNumbers.java
/*
Program 3 : Write a Program to print First 50 Odd Numbers .
Output: 1 3 5 . . . 99

*/

class OddNumber{
	
	public static void main(String args[]){
		
			System.out.println("First 50 odd numbers");
			for(int itr = 0 ; itr <= 100; itr++){

				if(itr%2!=0)
					System.out.printf("%d  ",itr);

			}
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/CPP/Oddnumbers.cpp
/*
Program 3 : Write a Program to print First 50 Odd Numbers .
Output: 1 3 5 . . . 99

*/

#include<iostream>

#define lowerlimit 0
#define upperlimit 100

int main(){
	
	printf("First 50 odd numbers \n");

	for(int itr = lowerlimit ; itr <= upperlimit; itr++){
		
		if(itr%2!=0)
			printf("%d  ",itr);

	}	
}	
